tests/functional/firefox/whatsnew/test_whatsnew_95.py

```diff
@@ -12,3 +12,10 @@
 def test_pocket_buttons_are_displayed(base_url, selenium):
     page = FirefoxWhatsNew95Page(selenium, base_url, locale="de").open()
     assert page.is_pocket_button_displayed
+
+
+@pytest.mark.skip_if_not_firefox(reason="Whatsnew pages are shown to Firefox only.")
+@pytest.mark.nondestructive
+def test_pocket_button_is_displayed_en(base_url, selenium):
+    page = FirefoxWhatsNew95Page(selenium, base_url, locale="en-US").open()
+    assert page.is_pocket_button_displayed
```

Comment:
Running this test locally fails for me since it's looking for `.mzp-c-button` as a descendant of `.wnp-main-cta`, but in the template you have both classes directly on the link element. You could either update the template to match what the test is looking for, or alternatively add a new selector in the test that is specific to the English WNP.